index.html

```diff
@@ -1,9 +1,26 @@
 <!DOCTYPE html>
 <hml>
+
 	<head>
 		<title>Introduction to the DOM</title>
 		<link rel="stylesheet" href="./styles.css">
 	</head>
+
 	<body>
+		<div class="container">
+			<div class="header">
+				<nav class="header--dropdown">Dropdown</nav>
+				<h1 class="header--h1">Header</h1>
+			</div>
+			<!-- header -->
+			<div class="section">
+				<h2 class="section--h2">Placeholder</h2>
+				<p>Bacon ipsum dolor amet ham pork loin frankfurter pastrami flank, chuck landjaeger rump pork chop shank biltong turkey t-bone pig. Jowl salami leberkas brisket capicola pork. Jerky brisket andouille, kielbasa pancetta pig tri-tip bresaola venison filet mignon. Bacon fatback biltong pork chop, beef ribs pig pastrami pork. Chicken biltong spare ribs swine, bresaola chuck fatback turducken filet mignon strip steak ribeye pork hamburger tenderloin. Jowl kielbasa cupim andouille hamburger.</p>
```

Comment:
If you created another div, I would recommend that you use `<div class="section__whateveryouwant">` etc.